decentraland/unity-explorer
# Fix: Handle RunConnectCycleStepAsync safer

### How to test

1. Santiy check on multiplayer features
2. Verify the multiplayer games are more stable with the queues 

### Changes
- Changed logic to prevent "flickering" between rooms
- Exceptions handling to prevent loop breaking
- Change of logic to remove extra UniTasks
